index.js

```diff
@@ -1365,20 +1316,24 @@
   //. //
   //. //   The value at position 1 is not a member of ‘NonZeroInteger’.
   //. ```
-  function NullaryType(name, url, test) {
-    function format(outer, inner) {
-      return outer(stripNamespace(name));
-    }
-    return new _Type(NULLARY, name, url, format, test, [], {});
+  function NullaryType(name) {
+    return function(url) {
+      return function(test) {
+        function format(outer, inner) {
```

Comment:
It looks like the `format` function could be shared between different applications of `NullaryType (name)`.

Reply:
Great point! I'll address this.

Reply:
:zap: